includes/Services/FlowService.php

```diff
@@ -68,6 +70,12 @@ private static function update_flow_data_recursive( $default_flow_data, $flow_da
 		$exception_list    = Flows::get_exception_list();
 		$updated_flow_data = array();
 		foreach ( $default_flow_data as $key => $value ) {
+			// To update the options with the recent version of flow data
+			if ( strcmp( 'version', $key ) === 0 ) {
+				$updated_flow_data[ $key ] = $value;
+				continue;
+			}
```

Comment:
Small question here, since this a recursive function, let's say we add a version to another key in flow data, will that version also get updated with the value from `default_flow_data`?